.prettierignore

```diff
@@ -47,3 +47,8 @@ packages/miniflare
 # so let's ignore all the c3 template files, exlcuding the c3.ts ones
 packages/create-cloudflare/templates/**/*.*
 !packages/create-cloudflare/templates/**/c3.ts
+
+# Some of the `@cloudflare/vitest-pool-workers` examples use `satisfies` as it
```

Comment:
I guess that as long as they are only in tests this is OK.
But I don't think our version of esbuild supports it, right?
So we need to be careful we don't encourage people to expect that it can be used in the Workers source.